requirements.txt

```diff
@@ -3,6 +3,6 @@ pymysql>=0.7.2
 pyparsing
 ipython
 tqdm
-networkx~=1.11
-pydotplus
+networkx
+pydot
```

Comment:
why `pydot` vs `pydotplus`?

Reply:
Networkx apparently flipped in 2.0; my memory from research a few months back was that pydot went unmaintained for a bit, pydotplus was a python 3 + added feature fork, subsequently pydot development picked up and supports pydot3 and pydotplus appears dormant.

1.11 docs:

https://networkx.github.io/documentation/networkx-1.11/reference/drawing.html#module-networkx.drawing.nx_pydot

```
Pydot

Import and export NetworkX graphs in Graphviz dot format using pydotplus.

Either this module or nx_agraph can be used to interface with graphviz.

See also

PyDotPlus
    https://github.com/carlos-jenkins/pydotplus
```

2.1 docs:

https://networkx.github.io/documentation/stable/reference/drawing.html#module-networkx.drawing.nx_pydot

```
Pydot

Import and export NetworkX graphs in Graphviz dot format using pydot.

Either this module or nx_agraph can be used to interface with graphviz.

See also

pydot
    https://github.com/erocarrera/pydot
Graphviz
```

Thanks for pointing it out - this reminds me I should double check rendering on Windows; my recollection from the initial testing of pydotplus to avoid the graphviz issues is that both seem to work but I'm not 100% on this.